.github/workflows/close-linked-issues-for-merged-prs.yml

```
@@ -0,0 +1,20 @@
+name: Close linked issues for merged pull requests
+
+on:
+  pull_request:
+    types: [closed]
+    branches:
+      - beta
+      - 'stable-*'
+      - 'epic/*'
+
+jobs:
+  closeIssueOnPrMergeTrigger:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Closes issues related to a merged pull request.
+        uses: ldez/gha-mjolnir@v1.5.0
```

Security wise it might be better to reference the sha instead of the version number.

```suggestion
        uses: ldez/gha-mjolnir@5574ed1f1151e4d2f11e3513cd85920a3a46bb7b # v1.5.0
```